telegram/_inline/inlinequeryresultmpeg4gif.py

```diff
@@ -68,6 +82,10 @@ class InlineQueryResultMpeg4Gif(InlineQueryResult):
         input_message_content (:class:`telegram.InputMessageContent`, optional): Content of the
             message to be sent instead of the video animation.
 
+    Raises:
+        :class:`ValueError`: If neither :paramref:`thumbnail_url` nor :paramref:`thumb_url` is
+            supplied or if both are supplied and are not equal.
```

Comment:
raises block should be after attributes

Reply:
for classis having the raises before the attributes makes sense to me … that's basically the end of the `__init__` method 🤔

Reply:
It comes after Args but before Attrs in `CommandHandler`, `ConversationHandler`, `ViaBot`. Is that wrong as well?

Reply:
ah hmm, for some reason I didn't remember ever seeing it like this, anyway after checking the docs this looks fine to me.